Updated windows installation docs

dominikl opened the pull request:
Updates the Windows installation documentation following @manics installation scripts.

Deployed on https://www.openmicroscopy.org/site/support/omero5.2-staging/sysadmins/windows/server-installation.html

A few notes:
- I don't know if/what kind of update the other windows pages need, e. g. https://www.openmicroscopy.org/site/support/omero5.2-staging/sysadmins/windows/service-tools.html
- @aleksandra-tarkowska : Is the remark about unsetting `omero.web.session_engine` and `omero.web.caches` still valid for Django 1.8? - clarified, still needed

**Depends on:** https://github.com/openmicroscopy/openmicroscopy/pull/4368

dominikl committed:
Updated windows installation docs

atarkowska commented on omero/sysadmins/windows/server-installation.txt:
```diff
@@ -124,6 +127,10 @@ The following are optional depending on what services you require:
       - Functionality
       - Downloads
 
+    * - Django (1.8.5)
```
Could we change to 1.8.x because it is LTS

atarkowska replied:
Django is not additional library really. I think is enough to mention pip install on IIS page.

atarkowska commented on omero/sysadmins/windows/server-installation.txt:
```diff
@@ -124,6 +127,10 @@ The following are optional depending on what services you require:
 
+    * - Django (1.8.5)
+      - OMERO.web
+      - `Django website <https://www.djangoproject.com/>`_
```
more appropriate link `Django 1.8 installation on Windows https://docs.djangoproject.com/en/1.8/howto/windows/`_

aleksandra-tarkowska commented:
@dominikl thanks for the update, could we also add under screenshot *IIS 7 Application Pool Advanced Settings* note block **if deploying on 64-bit Enable 32 bit applications setting set back to False**?

dominikl commented:
I'm currently using windows server 2012 for testing (as windows 8 desktop doesn't work that well), which has a little bit different GUI, so the screenshot would look a bit different, if that's not a problem I can add a screenshot. But shouldn't the **Enable 32 bit** part be set to **True**!?

dominikl commented:
And I'm not sure what to do with this "additional Libraries" section. Are these all libraries for OMERO.web? Then I could move the whole section to iis-install doc. If it's just meant for the OMERO.server Django would actually be an optional library. This section is really more confusing than helpful.

aleksandra-tarkowska commented:
> But shouldn't the Enable 32 bit part be set to True!?
I think If you install 64-bit python, it is no longer 32-bit application that's why should be false. On WindowsServer2012 with ``Python 2.7.9 (default, Dec 10 2014, 12:28:03) [MSC v.1500 64 bit (AMD64)]`` web is not working when processes in WOW64 are in 32bit mode.

aleksandra-tarkowska commented:
> Is the remark about unsetting ``omero.web.session_engine`` and ``omero.web.caches`` still valid for Django 1.8?

As we don't know which version people update from, I would leave it.

manics commented:
I thought we dropped support for 32 bit OMERO.server?

aleksandra-tarkowska commented:
> https://www.openmicroscopy.org/site/support/omero5.2-staging/sysadmins/windows/server-installation.html  I just ran through the doc again, now at the last step, starting omero.web in development mode. if you follow the instruction you get stuck there with error "django not installed"
so i'll just add the instruction `pip install -r share/web/requirements-py27-nginx.txt` there was well.

I renamed requirements fine to be more usable across the doc. Could you add `pip install -r share/web/requirements-py27-win.txt`

atarkowska commented on omero/sysadmins/windows/install-web/install-iis.txt:
```diff
@@ -9,17 +9,17 @@ IIS configuration (Windows)
 ::
 
-   $ pip install -r share/web/requirements-py26-iis.txt
+   $ pip install -r share/web/requirements-py27-iis.txt
```
`pip install -r share/web/requirements-py27-win.txt`

dominikl commented:
Just checked, one really doesn't need to enable the 32bit feature.

dominikl commented:
Ah sorry, no I understand what you meant @aleksandra-tarkowska , the screenshot is still showing the enabled 32bit option. I'll add a new screenshot.

jburel commented:
need to indicate that pillow version is 2.9.0
see https://github.com/ome/figure/pull/136

dominikl committed:
Some more windows doc tinkering:
- Updated IIS settings screenshot
- Added screenshot for var/log permissions
- Removed the additional python deps table (imho that was just
  confusing).

dominikl committed:
Added additional libs table again

joshmoore commented:
A passing comment having not looked at this deeply: *very nice*.

mtbc commented on omero/sysadmins/windows/install-web/install-iis.txt:
```diff
@@ -71,6 +71,12 @@ working with your IIS deployment.
    :align: center
    :alt: IIS 7 HTTP response header
 
+-  Make sure the OMERO's log directory has full read/write permissions for all users
```
"Make sure _that_ OMERO's ... " perhaps?

dominikl committed:
Updated screenshots, added links to IIS docs

dominikl commented:
**Review** last commit only, deployed on https://www.openmicroscopy.org/site/support/omero5.2-staging/sysadmins/windows/install-web/install-iis.html

hflynn commented:
Merge build running now, last commit will be built in about 10 mins

aleksandra-tarkowska commented:
@dominikl thanks looks good

rleigh-dundee commented:
Looks pretty good to me.

hflynn commented:
Waiting on code PR being merged before I merge this.

aleksandra-tarkowska commented:
@hflynn we also need to merge https://github.com/openmicroscopy/openmicroscopy/pull/4296. I wasn't able to fix Windows build as I have no rights on burbot and don't know anything about omego.

hflynn commented:
Both linked PRs merged so merging
